### ISSUE TITLE
Thoughts on what we should be doing

### ISSUE THREAD

**Issue by emilyamorris** [emilyamorris/mini-project-3#1]
Tree sap seems like the most interesting project to work on 

I think we should wrangle the data such that we can get isolate the days where the temperatures were in the correct range and look at the years to find trends. Maybe mapping elevations or tree locations? Getting the spatial data in is tricky...

**Comment by slednicky** [comment]
I agree! great. and we will ask in class about getting some of the missing data
